src/main/Java/jonathan/Jonathan.java

```diff
@@ -0,0 +1,257 @@
+                "       " + task + "\n" +
+                "     Now you have " + taskCounter + " tasks in the list.\n" +
+                "    ____________________________________________________________\n");
+    }
+
+    /*
+    Method to print all the Task, including the Task's status
+     */
+    public static void lookupTask(ArrayList<Task> list, int counter) {
+        System.out.println("    ____________________________________________________________\n" +
+                "    Here are the tasks in your list:");
+        for (int i = 0; i < counter; i++) {
+            System.out.println("    " + (i+1) + ". "  + list.get(i));
+        }
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    /*
+    Method to change the status of a Task
+     */
+    public static void changeStatus(Task task, boolean isDone) {
+        task.setDone(isDone);
+        // If the Task is mark done
+        if (isDone) {
+            System.out.println("    ____________________________________________________________\n" +
+                    "     Nice! I've marked this task as done:\n" +
+                    "       " + task + "\n" +
+                    "    ____________________________________________________________\n");
+        } else { // Unmark the Task
+            System.out.println("    ____________________________________________________________\n" +
+                    "     OK, I've marked this task as not done yet:\n" +
+                    "       " + task + "\n" +
+                    "    ____________________________________________________________\n");
+        }
+    }
+
+    /*
+    Method to export the data into .txt file
+     */
+    public static void saveData(ArrayList<Task> tasks) {
+        try {
+            File outFile = new File("data/jonathan.txt");
+            outFile.createNewFile();
+
+            FileWriter outFileWriter = new FileWriter(outFile);
+
+            for (int i = 0; i < tasks.size(); i++) {
+                Task task = tasks.get(i);
+
+                if (task instanceof Todo) {
+                    Todo todo = (Todo) task;
+                    String line = "T | " + todo.getIsDone() + " | " + todo.getDescription() + "\n";
+                    outFileWriter.write(line);
+                } else if (task instanceof Deadline) {
+                    Deadline deadline = (Deadline) task;
+                    String line = "D | " + deadline.getIsDone() + " | " + deadline.getDescription() +
+                            " | " + deadline.getBy() + "\n";
+                    outFileWriter.write(line);
+                } else if (task instanceof Event) {
+                    Event event = (Event) task;
+                    String line = "E | " + event.getIsDone() + " | " + event.getDescription() +
+                            " | " + event.getStart() + " | " + event.getEnd() + "\n";
+                    outFileWriter.write(line);
```

Comment:
Try to avoid the usage of magic strings as much as possible. For example: in this part of code, you are using the pipe symbol multiple times. You can consider saving it in a variable and use that instead.